packages/form-core/src/registration/FormControlsCollection.js

```diff
@@ -91,8 +93,9 @@
 export class FormControlsCollection extends Array {
   /**
    * @desc Gives back the named keys and filters out array indexes
+   * @return {string[]}
    */
-  keys() {
+  _keys() {
```

Comment:
Technically breaking change for `formElements`, but we assume no one uses formElemenets.keys() ? Not entirely sure if this is ok, but if we extend Array, TS will not let us override private `keys` method :(

Reply:
imho that's good 👍